<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShareListDialog</class>
 <widget class="QDialog" name="ShareListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>761</width>
     <height>671</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="UrlLine">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>分享链接：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="codeLine">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>提取码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>340</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="CustomListWidget" name="FileListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>750</width>
       <height>600</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="lastBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="DownloadProgressTableWidget" name="ProgessWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>750</width>
       <height>600</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>返回列表</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomListWidget</class>
   <extends>QListWidget</extends>
   <header>customlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>DownloadProgressTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">downloadprogresstablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
